src/LoggingExtras.jl

```diff
@@ -12,6 +12,17 @@ export TeeLogger, TransformerLogger, FileLogger,
     ActiveFilteredLogger, EarlyFilteredLogger, MinLevelLogger,
     DatetimeRotatingFileLogger, FormatLogger
 
+######
+# Re export Logging.jl from stdlib 
+# list is stable between julia 1.0 and 1.6
+# https://github.com/JuliaLang/julia/blob/release-1.6/stdlib/Logging/src/Logging.jl#L32-L46
+import Logging: @debug, @info, @warn, @error, @logmsg,
+    with_logger, current_logger, disable_logging, ConsoleLogger
+
```

Comment:
Should bne enough just to bring them in scope via `using`
```suggestion
using Logging
```

Reply:
Huh, OK. This does work - is the `import` only needed if you're going to add methods?

Reply:
correct